examples/bearer-auth-uri-format.yaml

```diff
@@ -0,0 +1,15 @@
+document:
+  dsl: 1.0.0-alpha1
+  namespace: examples
+  name: bearer-auth-uri-format
+  version: 1.0.0-alpha1
+do:
+  getPetById:
+    call: http
+    with:
+      method: get
+      endpoint:
+        uri: https://petstore.swagger.io/v2/pet/1
```

Comment:
`uri: https://petstore.swagger.io/v2/pet/{petId}` instead?

Reply:
no, this is valid to verify the URI that's not a template.